src/main/java/no/nav/veilarbvedtaksstotte/service/VedtakService.java

```diff
@@ -110,6 +119,11 @@ public DokumentSendtDTO fattVedtak(long vedtakId) {
         AuthKontekst authKontekst = authService.sjekkTilgangTilBrukerOgEnhet(AktorId.of(vedtak.getAktorId()));
         authService.sjekkErAnsvarligVeilederFor(vedtak);
 
+        if ("ISERV".equals(ofNullable(veilarbarenaClient.hentOppfolgingsbruker(Fnr.of(authKontekst.getFnr())))
+                .map(VeilarbArenaOppfolging::getFormidlingsgruppekode).orElse(""))) {
```

Comment:
Jeg har egentlig litt lyst til å dra dette ut en funksjon for at det skal være mer lesbart.

brukerErISERV()

Det ble en litt heftig if